ngrok-jetty/src/main/java/com/ngrok/jetty/NgrokEndpoint.java

```diff
@@ -29,7 +30,8 @@ public NgrokEndpoint(Scheduler scheduler, Connection conn) {
     }
 
     /**
-     * Throws an {@link UnsupportedOperationException}, as incomplete flush is not supported
+     * Throws an {@link UnsupportedOperationException}, as incomplete flush is not
+     * supported
```

Comment:
fix that docu

Reply:
ugh. my autoformatting isn't work *except* for javadoc apparently.
we're going with 120 character lines for docs?